lab2/index.html

```diff
@@ -0,0 +1,232 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans&display=swap">
+</head>
+
+<body>
+    <header class="upper">
+        <div class="upper__block">
+            <nav class="upper__nav">
```

Comment:
This block doesn't look like a navigation menu. By clicking on the icons there could show up just a popup. So please change tag name to `div`